scripts/docker/polkadot_injected_release.Dockerfile

```diff
@@ -22,12 +23,11 @@ RUN apt-get update && \
 	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
 		libssl1.1 \
 		ca-certificates \
-		curl \
 		gnupg && \
 	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
 # add repo's gpg keys and install the published polkadot binary
-	gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798 && \
-	gpg --export 9D4B2B6EB8F97156D19669A9FF0812D491B96798 > /usr/share/keyrings/parity.gpg && \
+	gpg --recv-keys --keyserver hkps://keys.mailvelope.com ${POLKADOT_GPGKEY} && \
```

Comment:
I think it could be useful to extract an ARG for the key server. Keyservers do go down and if the one we use here does go down, we will no longer be able to build the image. With an ARG, we can use the same default but that allows an external override (from CI for instance) without having to touch the image.